redirects.json

```
@@ -203,6 +203,10 @@
     "fromPath": "/*/upgrades/shard-chains/",
     "toPath": "/:splat/upgrades/sharding/"
   },
+  {
+    "fromPath": "/upgrades/shard-chains/",
+    "toPath": "/en/upgrades/sharding/"
```

```suggestion
    "toPath": "/en/roadmap/danksharding/"
```

Update with roadmap release